src/setup_payload/tests/TestQRCode.cpp

```diff
@@ -30,6 +30,11 @@
 #include <support/Span.h>
 #include <support/UnitTestRegistration.h>
 
+// Forwarding of "private" function tested
+namespace chip {
+std::string extractPayload(std::string inString);
```

Comment:
Can you put this in the corresponding header? It's not desirable to have to update declarations in source files other than the one corresponding to the definition.

Reply:
Fixed